apps/web/server/lib/setup/getServerSideProps.tsx

```diff
@@ -24,33 +31,36 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     };
   }
 
-  const deploymentKey = await prisma.deployment.findUnique({
-    where: { id: 1 },
-    select: { licenseKey: true },
+  const deployment = await prisma.deployment.findUnique({
+    where: {
+      id: 1,
+    },
   });
```

Comment:
I'm curious here. I'm assuming that for new self-hosted instances we would be creating a single deployment record. When would this happen?

Reply:
![image](https://github.com/user-attachments/assets/0e026da7-32b6-490e-a5cd-73de9b0400d0)
This happens as part of our migrations